docs/usage.md

```diff
@@ -186,7 +186,14 @@ You can choose different software to perform the imputation. In the following se
 [QUILT](https://github.com/rwdavies/QUILT) is an R and C++ program for rapid genotype imputation from low-coverage sequence using a large reference panel. The required inputs for this program are bam samples provided in the input samplesheet (`--input`) and a csv file with the genomic chunks (`--chunks`).
 
 ```bash
-nextflow run nf-core/phaseimpute --input samplesheet.csv --chunks chunks.csv --steps impute --tool quilt --outdir results --genome GRCh37 -profile docker
+nextflow run nf-core/phaseimpute --input samplesheet.csv --panel panel_haplegend.csv --chunks chunks.csv --steps impute --tool quilt --outdir results --genome GRCh37 -profile docker
+```
+
+The csv provided in `--panel` must contain at least four columns [panel, chr, hap, legend]. The first column is the name of the panel, the second is the chromosome, then the hap and legend files produced by `--steps panelprep` unique to each chromosome.
+
+```console
+panel,chr,vcf,index,hap,legend
+1000GP.s.norel,chr22,1000GP.s.norel_chr22.hap.gz,1000GP.s.norel_chr22.legend.gz
```

Comment:
You need to add two ",," or "1000GP.s.norel_chr22.vcf, 1000GP.s.norel_chr22.vcf.csi"